tst/Boolean/Assert-False.Tests.ps1

```diff
@@ -1,13 +1,13 @@
 Describe "Assert-False" {
-        It "Passes when given `$false" {
```

Comment:
the original text is imho just fine. The test could not be simpler, and there is no need to state the name of the assertion in the test description when we have it in describe.